### PR TITLE
Add Makefile command for building code

### PR DESCRIPTION
**Pull request by RonanMorgan**
I was a bit lost when I tried to build the code because of the separation between the lib and the app so I have added some Makefile commands in order to better reflect this architecture and I removed one Makefile file and the requirements.txt for simplification purpose

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,13 +19,29 @@ test:
...
+
+# update requirements.txt
+lock:
+	cd src; poetry lock
+	cd src; poetry export -f requirements.txt --without-hashes --output requirements.txt
+
 # Build the docker
-build:
+build-app:
 	docker build . -t pyronear/pyro-engine:python3.8.1-slim
```
Comment (MateoLostanlen) [inline Makefile:30]: why do you build two images here ?

**Review by MateoLostanlen — APPROVED** [review]
just one small question but it LGTM

File: Makefile (hunk 1 above)
Comment (RonanMorgan) [reply Makefile:30]: hum build-app is building one image, with the app contained in src/ and build-lib is building the library locally.
Maybe build-image and build-lib would be a better wording ?

Comment (MateoLostanlen) [reply Makefile:30]: Sorry, maybe it's just me, but I don't understand why we need these two lines, we can't just keep :

`docker build . -t pyronear/pyro-engine:latest`

Comment (RonanMorgan) [reply Makefile]: done @MateoLostanlen

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Build docker image
-        run: docker build . -t pyronear/pyro-engine:python3.8.1-slim
+        run: make build-app
       - name: Run docker container
-        run: docker run pyronear/pyro-engine:python3.8.1-slim python -c 'import pyroengine'
+        run: make run
+
```
Comment (RonanMorgan) [inline .github/workflows/docker.yml:18]:
```suggestion
```

**Review by MateoLostanlen — APPROVED** [review]
All good thanks

**Pull request merged**